workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/WorkerInspectorWindow.cs

```diff
@@ -34,6 +39,24 @@ private void OnInspectorUpdate()
             workerDetail.Update();
         }
 
+        private void HandleRightClick(MouseUpEvent evt)
+        {
+            if (evt.button != (int) MouseButton.RightMouse)
+            {
+                return;
+            }
+
+            var menu = new GenericMenu();
+
+            menu.AddItem(new GUIContent("Hide Empty Collections"), hideCollectionsIfEmpty, () =>
+            {
+                hideCollectionsIfEmpty = !hideCollectionsIfEmpty;
+                OnToggleHideCollections?.Invoke(hideCollectionsIfEmpty);
+            });
```

Comment:
I imagine this menu can be cached?

Reply:
Its only called after a right click. Also the `hideCollections` bool needs to be passed to the `AddItem` method for the context menu to indicate the value of the bool and there's no method to remove items.